deploy/examples/secure-query-with-keycloak-sidecar-values.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: v1
+kind: Jaeger
+  metadata:
+    name: jaeger
+    namespace: default
+  spec:
+    agent:
+      config: {}
+      options: {}
+      resources: {}
+    allInOne:
+      config: {}
+      options: {}
+      resources: {}
+    collector:
+      config: {}
+      options: {}
+      resources: {}
+    ingester:
+      config: {}
+      options: {}
+      resources: {}
+    ingress:
+      enabled: true
+      security: "oauth-proxy"
+    query:
+      oauthProxy:
+        options:
+          #        Client ID for OpenID server
+          - --client-id=jaeger-client
+          #        Client secret for OpenID server.
+          #        Confidential access type is for server-side clients that need to perform a browser login and require a client
+          #        secret when they turn an access code into an access token, (see Access Token Request in the OAuth 2.0 spec for
+          #        more details). This type should be used for server-side applications.
+          #        see https://www.keycloak.org/docs/6.0/server_admin/
+          - --client-secret=12345678-9abc-def0-1234-56789abcdef0
+          #        URL for OpenID autoconfiguration
+          - --discovery-url=https://keycloak.example.com/auth/realms/jaeger-realm
+          - --enable-default-deny=false
+          - --enable-json-logging=true
```

Comment:
Is this a good and secure default? Would most users want this set to `true` in their production environments? The same question applies to the other properties.